File: src/main/java/io/github/chaosdave34/kitpvp/kits/impl/elytra/KnightKit.java
package io.github.chaosdave34.kitpvp.kits.impl.elytra;

import io.github.chaosdave34.kitpvp.kits.ElytraKit;
import org.bukkit.Material;
import org.bukkit.enchantments.Enchantment;
import org.bukkit.inventory.ItemStack;

public class KnightKit extends ElytraKit {
    public KnightKit() {
        super("elytra_knight", "Knight", Material.STONE_SWORD);
    }

    @Override
    public ItemStack[] getInventoryContent() {
        ItemStack bow = new ItemStack(Material.BOW);
        bow.addEnchantment(Enchantment.ARROW_INFINITE, 1);
        bow.addEnchantment(Enchantment.ARROW_DAMAGE, 3);

        ItemStack stoneSword = new ItemStack(Material.STONE_SWORD);
        stoneSword.addEnchantment(Enchantment.DAMAGE_ALL, 1);

        return new ItemStack[]{
                bow,
                stoneSword,
        };
    }
}

File: src/main/kotlin/io/github/chaosdave34/kitpvp/kits/impl/elytra/KnightKit.kt
package io.github.chaosdave34.kitpvp.kits.impl.elytra

import io.github.chaosdave34.kitpvp.kits.ElytraKit
import org.bukkit.Material
import org.bukkit.enchantments.Enchantment
import org.bukkit.inventory.ItemStack

class KnightKit : ElytraKit("elytra_knight", "Knight", Material.IRON_SWORD) {

    override fun getInventoryContent(): Array<ItemStack?> {
        val bow = ItemStack.of(Material.BOW)
        bow.addEnchantment(Enchantment.INFINITY, 1)
        bow.addEnchantment(Enchantment.POWER, 3)

        val stoneSword = ItemStack.of(Material.IRON_SWORD)
        stoneSword.addEnchantment(Enchantment.SHARPNESS, 1)

        return arrayOf(
            bow,
            stoneSword,
        )
    }
}